core/src/test/java/org/apache/accumulo/core/conf/PropertyTest.java

```diff
@@ -152,7 +134,6 @@ private Property getDeprecatedProperty() {
 
   @Test
   public void testAnnotations() {
-    assertTrue(Property.GENERAL_VOLUME_CHOOSER.isExperimental());
```

Comment:
Oh, was this test failing?

Reply:
This shouldnt have been removed. I will undo in next commit. I thought at one point it was the only place `GENERAL_VOLUME_CHOOSER` was being used, but when I later cleared up my intellij issue I missed adding it back.